### PR TITLE
doc(release): Prepare release notes for oss 22.04.12

### PR DESCRIPTION
**Pull request by chgautier**
## Description

Release notes for oss 22.04.12

## Target version

- [ ] 21.10.x (staging)
- [x] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md [hunk 1]
```diff
@@ -16,6 +16,23 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
 
 ## Centreon Web
 
+### 22.04.12
+
+Release date: `March AA, 2023`
+
+#### Bug fixes
+
+- [CLAPI] Improved authentication by not checking password expiration policy for excluded users.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md:25]:
```suggestion
- [CLAPI] Fixed an issue to do not consider password expiration policy for excluded users.
```

File: versioned_docs/version-22.04/releases/centreon-core.md [hunk 2]
```diff
@@ -17,6 +17,23 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 22.04.12
+
+Release date: `March AA, 2023`
+
+#### Bug fixes
+
+- [CLAPI] Improved authentication by not checking password expiration policy for excluded users.
```
Comment (smau-centreon) [inline versioned_docs/version-22.04/releases/centreon-core.md:26]:
```suggestion
- [CLAPI] Fixed an issue to do not consider password expiration policy for excluded users.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md [hunk 3]
```diff
@@ -16,6 +16,23 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [Configuration] Fixed an issue in the recurrent downtimes form that caused service relations to be lost.
+- [Configuration] Fixed global configuration export button in banner with OIDC authentication.
+- [Core] Fixed blank page after login.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md:28]:
```suggestion
- [Core] Fixed a blank page issue after login.
```

File: versioned_docs/version-22.04/releases/centreon-core.md [hunk 4]
```diff
@@ -17,6 +17,23 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] Fixed an issue in the recurrent downtimes form that caused service relations to be lost.
+- [Configuration] Fixed global configuration export button in banner with OIDC authentication.
+- [Core] Fixed blank page after login.
```
Comment (smau-centreon) [inline versioned_docs/version-22.04/releases/centreon-core.md:29]:
```suggestion
- [Core] Fixed a blank page issue after login.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md [hunk 5]
```diff
@@ -16,6 +16,23 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [UI] Fixed an issue where a popup does not appear in legacy page graphs.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md:29]:
```suggestion
- [UI] Fixed an issue where a popup was not displayed in the graphs legacy page.
```

File: versioned_docs/version-22.04/releases/centreon-core.md [hunk 6]
```diff
@@ -17,6 +17,23 @@ If you have feature requests or want to report a bug, please go to our
...
+- [UI] Fixed an issue where a popup does not appear in legacy page graphs.
```
Comment (smau-centreon) [inline versioned_docs/version-22.04/releases/centreon-core.md:30]:
```suggestion
- [UI] Fixed an issue where a popup was not displayed in the graphs legacy page.
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

File: versioned_docs/version-22.04/releases/centreon-core.md (hunk 6 above)
Comment (chgautier) [reply versioned_docs/version-22.04/releases/centreon-core.md:30]: the issue is found in graphs of a certain legacy page, not a page where we only have graphs

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md (hunk 5 above)
Comment (chgautier) [reply i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md:29]: the issue is found in graphs of a certain legacy page, not a page where we only have graphs

**Review by sfarouq-ext — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by sfarouq-ext — APPROVED** [review]

**Pull request merged**